src/main/java/marites/Marites.java

```diff
@@ -0,0 +1,312 @@
+        printSeparator();
+    }
+    /**
+     * Prints an exit message.
+     */
+    private static void printExitMessage() {
+        System.out.println(EXIT_MESSAGE);
+        printSeparator();
+    }
+
+    /**
+     * Prints a feedback string from an executed command.
+     * @param feedback The feedback string to execute.
+     */
+    private static void showFeedback(String feedback) {
+        printSeparator();
+        System.out.print(feedback);
+        printSeparator();
+    }
+
+    /**
+     * Divides a line of user input into the command type, and the command.
+     * @param userInput The user's input
+     * @return A String[] with length 2. The first element is the command type,
+     *  while the second element is the command. If the command has no body
+     *  (e.g. list, unmark), this second element is an empty string.
+     */
+    private static String[] splitCommandTypeAndCommand(String userInput) {
+        String[] inputSplit = userInput.split("\\s", 2);
+        return new String[]{inputSplit[0].strip(),
+                (inputSplit.length > 1 ? inputSplit[1] : "").strip()};
+    }
+
+    /**
+     * Parses a user's command, and executes it.
+     * @param userInput A line of the user's input.
+     * @return Output that must be passed back to the user.
+     */
+    private static String processUserCommand(String userInput) {
+        String[] commandSplit = splitCommandTypeAndCommand(userInput);
+        String commandOutput;
+        String commandType = commandSplit[0], command = commandSplit[1];
+        switch (commandType) {
+        case COMMAND_EXIT:
+            commandOutput = executeExit();
+            break;
+        case COMMAND_LIST:
+            commandOutput = executeListTasks();
+            break;
+        case COMMAND_MARK:
+            commandOutput = executeSetTaskStatus(command, true);
+            break;
+        case COMMAND_UNMARK:
+            commandOutput = executeSetTaskStatus(command, false);
+            break;
+        case COMMAND_ADD_TODO:
+        case COMMAND_ADD_DEADLINE:
+        case COMMAND_ADD_EVENT:
+            // All 3 cases fallthrough here
+            commandOutput = executeAddTask(commandType, command);
+            break;
+        case COMMAND_DELETE:
+            commandOutput = executeDelete(command);
+            break;
+        default:
+            commandOutput = UNKNOWN_COMMAND_MESSAGE;
+            break;
+        }
+        return commandOutput;
+    }
+
+    /*
+        The following methods take in a String, denoting a command.
+        They return a String, denoting the feedback after executing the command.
+     */
+
+    /**
+     * Executes a list command.
+     * @return The list of tasks.
+     */
+    private static String executeListTasks() {
+        StringBuilder taskList = new StringBuilder();
+        for (int i = 1; i <= tasks.size(); ++i) {
+            String task = String.format(LIST_TASK_ITEM_FORMAT_STRING, i, tasks.get(i-1));
```

Comment:
**Coding standard violation:** Operators should be surrounded by a space character (e.g., `i - 1`).